cswinrt/strings/additions/Windows.Storage.Streams/WindowsRuntimeBufferExtensions.cs

```diff
@@ -154,10 +151,9 @@ public static void CopyTo(this IBuffer source, uint sourceIndex, byte[] destinat
             if (destination == null) throw new ArgumentNullException(nameof(destination));
             if (count < 0) throw new ArgumentOutOfRangeException(nameof(count));
             if (destinationIndex < 0) throw new ArgumentOutOfRangeException(nameof(destinationIndex));
-            if (source.Length == 0 && count == 0) return;
-            if (source.Capacity <= sourceIndex) throw new ArgumentException(global::Windows.Storage.Streams.SR.Argument_BufferIndexExceedsCapacity);
+            if (source.Capacity < sourceIndex) throw new ArgumentException(global::Windows.Storage.Streams.SR.Argument_BufferIndexExceedsCapacity);
             if (source.Capacity - sourceIndex < count) throw new ArgumentException(global::Windows.Storage.Streams.SR.Argument_InsufficientSpaceInSourceBuffer);
-            if (destination.Length <= destinationIndex) throw new ArgumentException(global::Windows.Storage.Streams.SR.Argument_IndexOutOfArrayBounds);
+            if (destination.Length < destinationIndex) throw new ArgumentException(global::Windows.Storage.Streams.SR.Argument_IndexOutOfArrayBounds);
```

Comment:
and shouldn't these be destination.Capacity, and not Length

Reply:
Here is documentation for this function, I think an array only has a Length property not Capacity.

https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.windowsruntime.windowsruntimebufferextensions.copyto?view=dotnet-plat-ext-3.1#System_Runtime_InteropServices_WindowsRuntime_WindowsRuntimeBufferExtensions_CopyTo_Windows_Storage_Streams_IBuffer_System_UInt32_System_Byte___System_Int32_System_Int32_